DataFormats/Detectors/ITSMFT/common/include/DataFormatsITSMFT/ClusterTopology.h

```diff
@@ -40,46 +36,38 @@ class ClusterTopology
   /// Standard constructor
   ClusterTopology(int nRow, int nCol, const unsigned char patt[Cluster::kMaxPatternBytes]);
 
+  /// Returns a specific byte of the pattern
+  unsigned char getByte(int n) const { return mPattern.getByte(n); }
   /// Returns the pattern
-  const std::array<unsigned char, Cluster::kMaxPatternBytes + 2>& getPattern() const { return mPattern; }
+  void getPattern(unsigned char destination[Cluster::kMaxPatternBytes + 2]) const { mPattern.getPattern(destination); }
```

Comment:
so what was the motivation for this change? (see above)

Reply:
I reverted to the `array<unsigned char, N>` version.
The change should have been only temporary, since I had some problems during the implementation of the last modifications to the code, but then I forgot to switch back. Now everything works fine.